unstructured/ingest/connector/azure_cognitive_search.py

```diff
@@ -120,3 +115,17 @@ def write(self, docs: t.List[BaseIngestDoc]) -> None:
                     ],
                 ),
             )
+
+    def write(self, docs: t.List[BaseIngestDoc]) -> None:
+        json_list: t.List[t.Dict[str, t.Any]] = []
+        for doc in docs:
+            local_path = doc._output_filename
+            with open(local_path) as json_file:
+                json_content = json.load(json_file)
+                for content in json_content:
+                    self.conform_dict(data=content)
+                logger.info(
+                    f"appending {len(json_content)} json elements from content in {local_path}",
+                )
+                json_list.extend(json_content)
+        self.write_dict(json_list=json_list)
```

Comment:
is this pattern generalizable so we don't have to redefine everywhere?

Reply:
Looking at it, other than the naming of it, the logic within each is highly specific to each connector and doesn't necessarily have to follow this pattern for any other downstream connectors.

Reply:
oh sorry, selected the wrong code for the comment here.